### PR TITLE
Update Feature Tables | Add M48 to 60 Family | Move Stop Detection

### PR DESCRIPTION
**Pull request by vertiq-jordan**
This first batch of RTD updates makes sure that names in the supported modules tables are generic (23-XX not 23-06), adds the M48 (60-08 660Kv) to the 60 family table, and moves stop detection out of the appendix, and into its own feature manual page.

### CODE REVIEW

File: docs/communication_protocols/dronecan_protocol.rst [hunk 1]
```diff
@@ -16,33 +16,13 @@ Module Support
...
-	+--------------+------------------------------------+
-	| Module       | DroneCAN Support                   |
-	+--------------+------------------------------------+
-	| Vertiq 81-08 | .. centered:: |:white_check_mark:| |
-	+--------------+------------------------------------+
-	| Vertiq 40-06 | .. centered:: |:white_check_mark:| |
-	+--------------+------------------------------------+
-	| Vertiq 23-06 | .. centered:: |:x:|                |
-	+--------------+------------------------------------+
+.. include:: ../manual/advanced_only_table.rst
```
Comment (vertiq-jordan) [inline docs/communication_protocols/dronecan_protocol.rst:19]: another common table. This time, we check off all of the advanced speed modules.

File: docs/communication_protocols/hobby_protocol.rst [hunk 2]
```diff
@@ -16,32 +16,12 @@ Module Support
...
-	+--------------+------------------------------------+
-	| Module       | Hobby Protocol Support             |
-	+--------------+------------------------------------+
-	| Vertiq 81-08 | .. centered:: |:white_check_mark:| |
-	+--------------+------------------------------------+
-	| Vertiq 40-06 | .. centered:: |:white_check_mark:| |
-	+--------------+------------------------------------+
-	| Vertiq 23-06 | .. centered:: |:white_check_mark:| |
-	+--------------+------------------------------------+
+.. include:: ../manual/all_checked_table.rst
```
Comment (vertiq-jordan) [inline docs/communication_protocols/hobby_protocol.rst:19]: another commonly used table, this one checks off everyone.

File: docs/index.rst [hunk 3]
```diff
@@ -78,6 +78,7 @@
    manual/manual_ifci_control
    manual/manual_underactuated_torque_correction
    manual/manual_buzzer_control
+   manual/manual_stop_detection
```
Comment (vertiq-jordan) [inline docs/index.rst:81]: move stop detection to its own home.

File: docs/manual/advanced_only_table.rst
Comment (vertiq-jordan) [inline docs/manual/advanced_only_table.rst:1]: common table that checks off only the advanced speed modules.

File: docs/manual/all_checked_table.rst
Comment (vertiq-jordan) [inline docs/manual/all_checked_table.rst:1]: common table with everyone checked off

File: docs/manual/fortiq_only_table.rst
Comment (vertiq-jordan) [inline docs/manual/fortiq_only_table.rst:1]: common table with only the fortiq checked

File: docs/manual/manual_appendix.rst [hunk 4]
```diff
@@ -7,29 +7,6 @@
 Appendix
 ***********************************************
 
-.. _appendix_stop_detection:
```
Comment (vertiq-jordan) [inline docs/manual/manual_appendix.rst:10]: move stop detection to its own feature manual page

File: docs/manual/manual_stop_detection.rst
Comment (vertiq-jordan) [inline docs/manual/manual_stop_detection.rst:1]: this is copy pasted from the content that was in the appendix. I've added our standard header with the title and supported modules.

File: docs/manual/manual_telemetry.rst [hunk 5]
```diff
@@ -18,35 +18,39 @@ Module Support
 Speed Modules
 **************
 
-.. table:: Speed Module Support for Telemetry
+.. table:: Module Support
```
Comment (vertiq-jordan) [inline docs/manual/manual_telemetry.rst:21]: tables with notes have to have their own special table. I still want it to match with everyone else...and it needs the 60.

File: docs/manual/manual_timeout.rst [hunk 6]
```diff
@@ -76,12 +68,12 @@ on a timeout if it was previously armed.** On a servo module, the module will co
 On modules with configurable timeout behavior, modules follow a multi-step timeout process when a timeout occurs, similar to the 
 :ref:`process that occurs when disarming <advanced_disarming_behavior>`.
 
-**This timeout process consists of 3 basic steps: The module switches how it is driving itself in order to try and come to a** :ref:`stop <appendix_stop_detection>` **, plays its timeout song as 
+**This timeout process consists of 3 basic steps: The module switches how it is driving itself in order to try and come to a** :ref:`stop <stop_detection>` **, plays its timeout song as
```
Comment (vertiq-jordan) [inline docs/manual/manual_timeout.rst:71]: use the new shortcut name

File: docs/modules/vertiq_60xx_family.rst [hunk 7]
```diff
@@ -15,6 +15,7 @@ Vertiq 60-XX Family
 
         "60-08", "150", "Speed", "Speed, Servo"
         "60-08", "300", "Speed", "Speed"
+        "60-08", "660", "Speed", "Speed"
```
Comment (vertiq-jordan) [inline docs/modules/vertiq_60xx_family.rst:18]: M48 goes in the table.

**Comment by iq-fred** [comment]
@vertiq-jordan Random thing, not sure if you're seeing it too, but on this branch I can't scroll down to the IQUART Client Table Reference: 
image

Not sure if its just we've hit some limit on scrolling on the sidebar or whatever? Or if its just my browser?

**Comment by vertiq-jordan** [comment]
> @vertiq-jordan Random thing, not sure if you're seeing it too, but on this branch I can't scroll down to the IQUART Client Table Reference: image
> 
> Not sure if its just we've hit some limit on scrolling on the sidebar or whatever? Or if its just my browser?

I think it has something to do with the branch name itself. The same thing is happening a bit on Luca's docs:
image
You can see the bottom of the label is cut off, but not the whole thing. Compared to my branch, his Read the Docs dropdown branch selector guy is smaller.

**Review by iq-fred — APPROVED** [review]

**Pull request merged**